docs/admin/resourcequota/index.md

```diff
@@ -52,8 +52,8 @@ Resource Quota is enforced in a particular namespace when there is a
 
 ## Compute Resource Quota
 
-The total sum of [compute resources](/docs/user-guide/compute-resources) requested by pods
-in a namespace can be limited.  The following compute resource types are supported:
+The total sum of [compute resources](/docs/user-guide/compute-resources) and [storage resources](/docs/user-guide/persistent-volumes)
```

Comment:
Can be limited, as in "these things are often in short supply", or you can now impose admin-specified limits on compute resources?

If the latter, I'd rephrase to "You can limit the total sum of compute resources and storage resources that can be requested in a given namespace."